src/dmd/target.h

```diff
@@ -74,6 +74,26 @@ struct TargetObjC
 
 struct Target
 {
+    typedef unsigned char OS;
+    enum
+    {
+        /* These are mutually exclusive; one and only one is set.
+         * Match spelling and casing of corresponding version identifiers
+         */
+        TargetOS_linux        = 1,
+        TargetOS_Windows      = 2,
+        TargetOS_OSX          = 4,
+        TargetOS_OpenBSD      = 8,
+        TargetOS_FreeBSD      = 0x10,
+        TargetOS_Solaris      = 0x20,
+        TargetOS_DragonFlyBSD = 0x40,
```

Comment:
Don't need the Target prefix here, just OS_xxx

Reply:
done